src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
```

Comment:
since we have only one `html` tag we can avoid adding a class name and use tag selector
```suggestion
<html lang="en">
```